sonar-text-plugin/src/main/java/org/sonar/plugins/secrets/configuration/model/matching/filter/PostModule.java

```diff
@@ -20,14 +20,15 @@
 
 package org.sonar.plugins.secrets.configuration.model.matching.filter;
 
+import java.util.List;
 import javax.annotation.Nullable;
 
 public class PostModule {
 
   @Nullable
   private HeuristicsFilter heuristicFilter;
   @Nullable
```

Comment:
Will it still deserialize empty lists as nulls or can Jackson be set to make them empty collections?

Reply:
It will be deserialized as null. As we have this behavior in multiple places, I will create a ticket to change it in multiple places.